addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -277,6 +277,19 @@ export class OdooEditor extends EventTarget {
          * ArrowUp/ArrowDown, between blocks, if there is no existing <p>
          */
         this._navigationNode = null;
+        /**
+         * Element that will stand out in the editor. If this property is set,
+         * subsequent @see 'selectionchange' or @see _onKeyDown calls will
+         * decide if the element should be removed or unselected. Its main
+         * purpose is to handle the deletion of contenteditable=false elements
+         * (which do not have a caret). Such elements can be "selected" via this
+         * property with @see oDeleteBackward when the selection is at the start
+         * of the next (editable) sibling. The editable will be removed if it is
+         * empty, and the uneditable will be selected. The element is unselected
+         * when the selection change, or upon keydown events.
+         */
+        this._uneditableSelection = null;
+        this._lockedUneditableSelection = false;
```

Comment:
No problemo, I'm still interested in your opinion regarding the concept.
I find it is a pretty clean way to remove blocks as it shows the user what may be removed. But it is indeed a lot of code for a small feature.

The only thing that I really want to be able to do is to remove an empty block between 2 contenteditable=false.

My previous iteration for this was to use CTRL+BACKSPACE, but @Goaman was against it because not everyone is aware of the difference between CTRL+BACKSPACE and BACKSPACE, and proposed that it should be done with BACKSPACE. 

But if we do BACKSPACE in the current editor in an empty block after a contenteditable=FALSE, it is the contenteditable=FALSE that is removed, not the empty block. 

Hence this solution which delete the empty block, and select the contenteditable=FALSE to be able to remove it or not if we do BACKSPACE again (or other things)